tests/tfm/tfm_regression_test/testcase.yaml

```diff
@@ -24,3 +24,11 @@ tests:
 
```

Comment:
@magnev Should we consider both enabled, and both disabled also?

Reply:
Yes, that would guarantee that we test all backend variants. What do you think @frkv ?